client/routes/domain/workflow-list.vue

```diff
@@ -185,13 +188,18 @@ export default {
         return null;
       }
 
+      if (queryString) {
+        return {
+          queryString: queryString.trim(),
+        };
+      }
+
```

Comment:
1. & 6.

Reply:
> 1. Text fields no longer try to trim themselves when entering whitespace (spacebar). Unless your copy/pasting, this is extremely annoying to use. The trim will occur when trying to make an API call instead so you effectively get the end result without the annoyance.

Thank you so much.  This has been putting what little sanity I have left in peril.

![](https://www.mememaker.net/static/images/memes/3837418.jpg)